### DISCUSSION TITLE
Date column value is being inserted as a timestamp string

### DISCUSSION THREAD

**Discussion by dreinon** [PostgREST/postgrest#2062] [Q&A] [answered]
When I try an insert where a column is of type date, with format
```json
{
    "date_column": "YYYY-MM-DD"
}
```
a string with format "YYYY-MM-DDT00:00:00.000Z" is added to the db instead of just the date.

**Comment by dreinon** [comment]
Any help with this please? Should I open an issue?

**Comment by wolfgangwalther** [comment, marked as answer]
You'll need to provide more details:
- add a very basic example DDL statement to create the table, so we can reproduce easily
- add the exact request you're making to this table
- tell us where you get that timestamp string as a response. From PostgREST? Or directly via SQL? Which request/query?

**Reply by dreinon** [reply]
It's an issue of supabase, instead of postgrest. Sorry.
